### DISCUSSION TITLE
Accessibility / Wheelchair routing?

### DISCUSSION THREAD

**Discussion by simonbaronti** [valhalla/valhalla#4788] [Q&A] [answered]
Hello there, 

We're trying to implement an "accessibility" toggle within our map options that will allow users to only show routes that are only suitable for wheels. Of course, this is set within the Wheelchair Access Field on OSM, however we cannot find a solution to pull this info in.

Am I missing something, or does anyone have any suggestions on being able to provide an solution for "accessibility" purposes.

Thanks in advance :)

**Comment by chrstnbwnkl** [comment, marked as answer]
There's a `wheelchair` option for pedestrian costing. Here's an example:

```
{
	"costing": "pedestrian",
	"costing_options": {
		"pedestrian": {
			"step_penalty": 0,
					"type": "wheelchair"
		}
	},
	"locations": [
		{
			"lon": 6.9619739055633545,
			"lat": 50.9408426595798,
			"type": "break"
		},
		{
			"lon": 6.961179971694947,
			"lat": 50.94086632125546,
			"type": "break"
		}
	],
	"directions_options": {
		"units": "kilometers"
	},
	"id": "valhalla_directions"
}
```

This seems to currently toggle a couple of switches in costing, e.g.

  - max out the steps penalty
  - reduce the speed
  - disallow edges with no wheelchair access 

I can't seem to find too many issues related to this, possibly because it's not really documented. Would be interesting to get some more feedback on this!

**Reply by simonbaronti** [reply]
Fantastic, thanks @chrstnbwnkl  - will feedback once we do more testing, but this is a great help :)
